### PR TITLE
Adding Datasources to call User and Trainee API

### PR DESCRIPTION
**Pull request by nehasoniengg**

### CODE REVIEW

File: src/services/trainee/trainee.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  constructor() {
+    super();
+    this.baseURL = configuration.service_url
+  }
+  willSendRequest(request) {
+    request.headers.set('Authorization', this.context.token);
+  }
+  async getTraine() {
+    const data = await this.get(`trainee`);
+    console.log("data:::::::::", data);
```
Comment (pinku4444) [inline src/services/trainee/trainee.js]: please remove console log

File: src/module/trainee/query.js [hunk 2]
```diff
@@ -1,26 +1,31 @@
...
-    },
-    {
-        name: 'trainee3',
-        email: 'trainee3@successive.tech',
-    },
-    {
-        name: 'trainee4',
-        email: 'trainee4@successive.tech',
-    },
+// const trainee = [
```
Comment (pinku4444) [inline src/module/trainee/query.js]: Please remove commented code

**Review by pinku4444 — COMMENTED** [review]
Please fix all issue.

File: src/services/trainee/trainee.js (hunk 1 above)
Comment (nehasoniengg) [reply src/services/trainee/trainee.js]: Done

File: src/module/trainee/query.js (hunk 2 above)
Comment (nehasoniengg) [reply src/module/trainee/query.js]: Done.

**Pull request merged**